src/Directory.Build.props

```diff
@@ -21,7 +21,7 @@
     <CreateHardLinksForAdditionalFilesIfPossible>true</CreateHardLinksForAdditionalFilesIfPossible>
     <CreateHardLinksForPublishFilesIfPossible>true</CreateHardLinksForPublishFilesIfPossible>
 
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>preview</LangVersion>
```

Comment:
Do we really want to merge this into ' master'?

Reply:
We don't ;)